ceno_cli/src/commands/common_args/cargo.rs

```diff
@@ -38,11 +38,11 @@ pub struct CargoOptions {
 }
 
 /// Package Selection:
-///   -p, --package [<SPEC>]  Package with the target to run
+///   --package [<SPEC>]  Package with the target to run
 #[derive(Clone, Args)]
 pub struct PackageSelection {
     /// Package with the target to run
-    #[arg(short = 'p', long)]
+    #[arg(long)]
```

Comment:
this "short" conflict with another "platform". 

https://github.com/scroll-tech/ceno/blob/75d75ea49c5a0c9a9f4383561eaa95cf003b7b19/ceno_cli/src/commands/common_args/ceno.rs#L27

Reply:
prefer to make platform flag only available at `--platform` to keep compatible with cargo cli.

Reply:
could you elaborate more on the comment?
originally both "--package", "--platform" conflict on short "-p" 
I think we should let "-p" works on platform, while leave "--package" will full name

Reply:
I prefer to let `-p` used by `--package`, which makes the behavior same as `cargo` itself.